### PR TITLE
don't depend on theforeman/stories

### PR DESCRIPTION
**Pull request by evgeni**
this was dropped from core in https://github.com/theforeman/foreman/commit/5e7343ecdad7632a94efa26fcba84889e66b06b4

### CODE REVIEW

**Comment by evgeni** [comment]
:eyes: @Ron-Lavi @ezr-ondrej

File: package.json [hunk 1]
```diff
@@ -14,7 +14,6 @@
     "@theforeman/builder": "^8.4.1",
     "@theforeman/eslint-plugin-foreman": "^8.4.1",
     "@theforeman/find-foreman": "^8.4.1",
-    "@theforeman/stories": "^8.4.1",
     "@theforeman/test": "^8.4.1",
     "@theforeman/vendor-dev": "^8.4.1",
```
Comment (Ron-Lavi) [inline package.json:18]: @evgeni can you try to change the packages accepted version to ">= 8.4.1" ?

Comment (evgeni) [reply package.json]: done, and now?

Comment (Ron-Lavi) [reply package.json]: thanks, now `npm test -- -u` to fix the tests snapshots

Comment (evgeni) [reply package.json]: I don't have a dev setup locally where I could run that. Sorry.
(and my change should not have changed any snapshots?)

Comment (ezr-ondrej) [reply package.json]: I'm guessing it happens through upgrade of an package, so it's probably broken in main (that's why I hate snapshoting btw :stuck_out_tongue: )

Comment (ezr-ondrej) [reply package.json]: I'd love to fix the snapshots, but no dev setup here either :(

**Review by Ron-Lavi — APPROVED** [review]
Changes looks good and test failures aren't related and could be fixed in a follow-up

**Comment by Ron-Lavi** [comment]
@ezr-ondrej could you give me merge rights here? I could check on this repo from time to time :P

**Comment by ezr-ondrej** [comment]
Done, I believe it's going to be bit better in your hands then in mine! Thanks! :)

**Pull request merged**